ui/src/components/SubPlayPage.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import DrawingCanvas from "./DrawingCanvas";
+
+const SubPlayPage = ({number_truth = "0"}) => {   
+    const [canvasURI, setCanvasURI] = useState("")  
+    const [number_result, SetNumber] = useState(0);
```

Comment:
I think it's better to have this const [numberResult, SetNumberResult] = useState(0);